operator/v1/types_console.go

```diff
@@ -18,7 +18,8 @@ type ConsoleSpec struct {
 	OperatorSpec `json:",inline"`
 	// customization is used to optionally provide a small set of
 	// customization options to the web console.
-	Customization ConsoleCustomization `json:"customization,omitempty"`
+	// +optional
```

Comment:
@benjaminapetersen also take a look here.

Reply:
Same as above, I am good with this behavior, thanks!